worker/actionpruner/worker.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package actionpruner
+
+import (
+	"time"
+
+	"github.com/juju/errors"
+	"gopkg.in/juju/worker.v1"
+
+	"github.com/juju/juju/api/action"
+	"github.com/juju/juju/api/base"
+	"github.com/juju/juju/environs/config"
+	"github.com/juju/juju/worker/catacomb"
+	"github.com/juju/juju/worker/pruner"
+)
+
+// Worker prunes status history records at regular intervals.
+type Worker struct {
+	pruner.PrunerWorker
+}
+
+func NewFacade(caller base.APICaller) pruner.Facade {
+	return action.NewFacade(caller)
+}
+
+type configUpdater struct {
+	config *config.Config
+}
+
+// Indicates that the collection size is specified by action
```

Comment:
The Go standard for doc comments is to have them start with the name of the thing the comment is attached to. (Which feels a bit repetitive to me but better to be consistent.)
https://blog.golang.org/godoc-documenting-go-code